### DISCUSSION TITLE
comet-land dev 실행 시 port already in use 오류

### DISCUSSION THREAD

**Discussion by hanseulhee** [hyesungoh/comet-land#102] [Q&A] [answered]
comet-land dev 실행 시 port already in use 오류가 뜹니다.

**Comment by hyesungoh** [comment, marked as answer]
정상적으로 dev 서버가 종료되지 않아 발생한 문제입니다.

```bash
npx kill-port 3000
npx kill-port 3001
```

[kill-port](https://www.npmjs.com/package/kill-port)를 이용해 포트를 내린 후 실행하시면 정상적으로 실행되실겁니다 :D

**Reply by hanseulhee** [reply]
감사합니당 😄
